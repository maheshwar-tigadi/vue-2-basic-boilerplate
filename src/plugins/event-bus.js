import Vue from 'vue'

export const eventBus = new Vue(); // creating an event bus.
