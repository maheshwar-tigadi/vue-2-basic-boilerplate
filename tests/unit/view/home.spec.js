import { shallowMount } from '@vue/test-utils'
import home from '@/views/home'
import { i18n } from '@/plugins/vue-i18n';

describe('home.test.js', () => {
    let cmp;
    beforeEach(() => {
        cmp = shallowMount(home, {
            i18n
        })
    })

    it('has the expected html structure', () => {
        expect(cmp.element).toMatchSnapshot()
    })
})