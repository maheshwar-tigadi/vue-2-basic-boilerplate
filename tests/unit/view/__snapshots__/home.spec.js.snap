// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home.test.js has the expected html structure 1`] = `
<v-layout-stub>
  <v-card-stub
    contextualstyle="dark"
  >
    <span>
      
      Welcome!
    
    </span>
     
    <div>
      <p>
        
        Get started with the Vue 2 boilerplate
      
      </p>
      
      
    
    </div>
     
    <div>
      
      Made with love by webtutorials.dev
    
    </div>
  </v-card-stub>
</v-layout-stub>
`;
